metecho/api/models.py

```diff
@@ -671,6 +678,35 @@ def get_base(self):
     def get_head(self):
         return self.branch_name
 
+    def try_to_notify_assigned_user(self):
+        # This takes the tester (a.k.a. assigned_qa) and sends them an
+        # email when a PR has been made.
+        assigned = self.assigned_qa
+        id_ = assigned.get("id") if assigned else None
+        sa = SocialAccount.objects.filter(provider="github", uid=id_).first()
+        user = getattr(sa, "user", None)
+        if user:
+            from .jobs import get_user_facing_url
```

Comment:
Should this be in a utilities module (but better named) above both `models` and `jobs`? It's not a job itself, so I don't think it needs to be in `jobs`.

Reply:
Yeah, I had a similar thought -- you wanna go ahead and clean it up?

Reply:
Will do.